src/styles/components/data-table.scss

```diff
@@ -90,20 +95,47 @@
     }
   }
 
-  &__row {
-    &--selected {
-      background-color: $colour-selected;
-    }
-  }
-
   // table body cell, without any specific modifier applied
   & td {
     padding: 0.5rem;
-    vertical-align: top;
     border: 0;
+    vertical-align: top;
     border-bottom: 1px solid $colour-sky-white;
   }
 
+  & .checkbox-cell {
+    position: relative;
+    padding: 0;
+
+    & > input[type='checkbox'] {
+      margin: 0.9em 0.75ch 0 1.25ch;
+
+      & + label {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        margin: 0;
+        border-left: 1ch solid $colour-sea-blue;
+        opacity: 0;
+
+        transition: opacity ease-in-out 0.1s;
+      }
+
+      &:hover,
+      &:focus {
```

Comment:
If you de-select a checkbox, I would expect the border to go all together but it stays (semi opaque) unless you click outside, I guess because the row still has the focus.

Reply:
Are you asking to change the focus/hover state? And with what?